src/components/Sum.jsx

```diff
@@ -1 +1,21 @@
-// Don't forget to add default values an check types of props
+import { number } from 'prop-types';
+import React from 'react';
+
+const Sum = ({
+  a,
+  b,
+}) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+);
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
+
+Sum.propTypes = {
+  a: number,
+  b: number,
+};
```

Comment:
I'm not sure, but propTypes is spelled like this:
```suggestion
Sum.propTypes = {
  a: PropTypes.number,
  b: PropTypes.number,
};
```

Reply:
Dmytro is right